### PR TITLE
fix(OverscrollColorProvider): avoid re-mounts

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: src/overscroll-color-context.tsx [hunk 1]
```diff
@@ -6,51 +6,50 @@ import {isInsideNovumNativeApp, getPlatform} from './utils/platform';
...
-                                    500 +
-                                    (isInsideNovumNativeApp(platformOverrides)
-                                        ? 0
-                                        : theme.dimensions.headerMobileHeight),
-                                left: 0,
-                                marginTop: -500,
-                                transform: 'translate3d(0,0,0)',
-                            }}
-                        />
+                        {isTabletOrSmaller ? (
```
Comment (pladaria) [inline src/overscroll-color-context.tsx:27]: I'd use:
```js
isTabletOrSmaller && (
   ...
```

**Review by pladaria — APPROVED** [review]
Thanks!

File: src/__tests__/overscroll-color-context-test.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const DummyComponent = ({callback}: {callback: jest.Mock}) => {
+    React.useEffect(() => {
+        callback();
+    }, [callback]);
+
+    return <span>dummy component</span>;
+};
+
+test('overscroll does not cause a remount of its children', async () => {
+    resizeWindow(640, 480);
```
Comment (atabel) [inline src/__tests__/overscroll-color-context-test.tsx:23]: if you're not changing this, how are you trying to replicate the issue in this tests? Is this test failing without your changes?

Comment (sergivillar) [reply src/__tests__/overscroll-color-context-test.tsx:23]: yep, it is failing without my changes, this issue happens without the need of change window size, but in a small window size (mobile or tablet)

**Review by atabel — APPROVED** [review]

**Pull request merged**